openvino_xai/explainer/explanation.py

```diff
@@ -149,6 +151,90 @@ def save(self, dir_path: Path | str, name: str | None = None) -> None:
             image_name = f"{save_name}_target_{target_name}.jpg" if save_name else f"target_{target_name}.jpg"
             cv2.imwrite(os.path.join(dir_path, image_name), img=map_to_save)
 
+    def plot(
+        self, targets: np.ndarray | List[int | str] | None = None, backend="matplotlib", threshold: int = 24
+    ) -> None:
+        """
+        Plots saliency maps using the specified backend.
+
+        This function plots available saliency maps using the specified backend. Additionally, target classes
+        can be specified by passing a list or array of target class indices or names. If a provided class is
+        not available among the saliency maps, it is omitted.
+
+        Args:
+            targets (np.ndarray | List[int | str] | None): A list or array of target class indices or names to plot.
+                By default, it's None, and all available saliency maps are plotted.
+            backend (str): The plotting backend to use. Can be either 'matplotlib' (recommended for Jupyter)
+                or 'cv' (recommended for Python scripts). Default is 'matplotlib'.
+            threshold (int): Max number of images to plot. 50 by default to avoid memory issues.
```

Comment:
`max_plots`, `max_num_plots` might be more descriptive.
I would have been confused whether it's confidence threshold.